src/js/utils/storage.js

```diff
@@ -2,25 +2,24 @@ export let idbKeyval = (() => {
   let db;
 
   function getDB() {
-    if (!db) {
-      db = new Promise((resolve, reject) => {
-        const openreq = indexedDB.open('svgo-keyval', 1);
+    if (db) return db;
```

Comment:
So db will never be cached?

Reply:
I'd say the code is fine as is. Extra nesting is not always bad.

Reply:
I generally prefer early returns, but yeah, when it comes to caching I prefer:

```js
function whatever() {
  if (notCached) addToCache();
  return getFromCache();
}
```

To:

```js
function whatever() {
  if (cached) return getFromCache();
  addToCache();
  return getFromCache();
}
```

The latter feels more prone to error, like changing how one of the returns works, or forgetting the `addToCache` bit (which is what's happening in this PR).